src/main/resources/jnt_categorizing/html/categorizing.jsp

```diff
@@ -51,7 +51,10 @@
 
                     var catDiv = $('<div></div>').attr('id','category'+catToAddUuid).attr('style','display:inline');
                     var catDisplay = jQuery('<span class="categorizeditem">' + $("#category").val() + '</span>');
-                    var catLinkDelete = $('<a></a>').attr('onclick','deleteCategory(\''+ catToAddUuid +'\')').attr('class','delete').attr('href','#');
+                    var catLinkDelete = $('<a></a>').attr('class','delete').attr('href','#');
+                    catLinkDelete.click(function() {
```

Comment:
I am not familiar with the implementation detail, but it this sufficient to do this (in respect to the CSP issue)?

Reply:
It attaches a listener instead of an onclick inline attribute so that means that inline attribute is removed, which is what the ticket is asking for.